### PR TITLE
Fix pe-d bugs in UG and DG

### PR DESCRIPTION
**Pull request by Rachelcoll**
Fixes issues 
- #275 
- #272
- #269 
- #265
-  #262
-  #237 
- #233 
- #261 
- #230 
- #228 
- #224 
- #221
-  #213

- Add `Feature Enhancement` in DG
- Update sequence and class diagram in DG, remove arrowheads in aggregation and composition relationship in DG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -121,6 +121,11 @@ A person is uniquely identified by his/her **telegram handle**
 A person can have any number of roles (including 0)
 </div>
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**<br/>
+Member is a special type of contact in CCAConnect, attendance related command will only work for members.
+If you are adding a new member with `add` command, please add "member" (case-insensitive) as a role for the new contact.
+</div>
```
Comment (shizy) [inline docs/UserGuide.md:144]: Could make it clearer:
`Member` is a **special reserved role** in CCAConnect. Attendance related commands will only work for contacts that have the `Member` role.<br/>
When adding/editing a person with the role of `Member`, the spelling of `Member` in `r/Member` is case-insensitive. E.g. `r/mEmBER` will also set the person to be a `Member`

**Review by shizy — CHANGES_REQUESTED** [review]
Maybe one change for now

File: docs/UserGuide.md (hunk 1 above)
Comment (Rachelcoll) [reply docs/UserGuide.md:144]: Thanks, already updated.

**Review by weiliann — APPROVED** [review]

**Review by shizy — APPROVED** [review]
LGTM

**Pull request merged**